model-optimizer/extensions/front/tf/sparse_to_dense.py

```diff
@@ -0,0 +1,67 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import numpy as np
+
+from extensions.ops.Cast import Cast
+from extensions.ops.scatternd import ScatterNDUpdate
+from mo.front.common.replacement import FrontReplacementOp
+from mo.graph.graph import Node, Graph, rename_nodes
+from mo.ops.broadcast import Broadcast
+from mo.ops.const import Const
+
+
+class SparseToDense(FrontReplacementOp):
```

Comment:
Please, change the class name. This class can be confused with the operation SparseToDense. Consider name like: `SparseToDenseTransformation`

Reply:
Fixed. Rename to `SparseToDenseReplacer`